### DISCUSSION TITLE
[#1065637] Paste mode gizmos to move

### DISCUSSION THREAD

**Discussion by FrederoxDev** [Mojang/minecraft-editor#126] [Feedback]
Original discord conversation: https://discord.com/channels/523663022053392405/1084090299120373760/1121808779370049667

![](https://media.discordapp.net/attachments/1084090299120373760/1121808778984161400/image.png?width=1141&height=606)

It would make more sense for the Gizmo to be attached to the pasting selection box and not the original.

**Comment by minecraftchloe** [comment]
Thank you!!
